src/sign-in-webapp/README.md

```diff
@@ -0,0 +1,106 @@
+  1──────────────────────────────────────────────►│                            │                                         │                            │
+  │            browser redirection                │  ASP.NET Core 6            │   https://graph.microsoft.com/beta/me   │   Microsoft 365            │
+  │   2───────────────────────────────────────────┤                            ├─6─────────────────────────────────────► │                            │
+  │   │             https://<fqdn>/signin-oidc    │  Web App                   │   Authorization Bearer 1NS...           │   Graph Api                │
+  │   │   5──────────────────────────────────────►│                            │                                         │                            │
+  │   │   │                                       │                            │                                         │                            │
+  │   ▼   │                                       └────────────────────────────┘                                         └────────────────────────────┘
+┌─┴───────┴─────────────────┐
+│                           │
+│                           │
+│                           │
+│   client (browser)        │
+│                           │
+│                           │
+│                           │
+└─────┬─────────────────────┘
+      │   ▲
+      │   │                                       ┌────────────────────────────┐
+      │   │                                       │                            │
+      │   │                                       │                            │
+      │   │                                       │   Azure AD                 │
+      │   │         Security Token (claims)       │                            │
+      │   4───────────────────────────────────────┤   STS                      │
+      │                                           │                            │
+      3──────────────────────────────────────────►│                            │
+            https://login.microsoftonline.com/    └────────────────────────────┘
+
+Scenario:
+
+A protected web app allows users to sign in, it enables the possiblity of acquiring and validating their tokens.
+
+Later the web app can make calls to the Microsoft 365 Graph API on behalf of signed-in user.
+```
+
+:link: For more information about how to proctect your projects, please let's take a look at https://docs.microsoft.com/en-us/azure/active-directory/develop/sample-v2-code. To know more about how this sample has been generated, please visit https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/?view=aspnetcore-6.0
+
+## Prerequisites
+
+1. An Azure Active Directory (Azure AD) tenant. You can [open an Azure account for free](https://azure.microsoft.com/free) to get an Azure AD instance.
+1. [Download .NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
+1. Install [JQ](https://stedolan.github.io/jq/download/)
```

Comment:
Avoid external dependencies whenever possible. Can you use JMESPath queries in your `az` CLI commands instead of JQ?

[How to query Azure CLI command output using a JMESPath query](https://docs.microsoft.com/cli/azure/query-azure-cli#get-a-single-value)

Reply:
thanks for the feedback @mmacy, 

done | addressed from [4008b](https://github.com/Azure-Samples/ms-identity-docs-code-csharp/pull/4/commits/4008be697bb14b9115bd3e842c6158ddb906e9a8)

Reply:
Looks good.